tests/data/perl/report.xml

```diff
@@ -1,12 +1,18 @@
 <?xml version='1.0' encoding='utf-8'?>
 <testsuites>
-  <testsuite name="t/00_compile.t" errors="0" failures="0" skipped="0" tests="1" time="0.0876882076263428">
-    <system-out>ok 1 - use Example;
-1..1
+  <testsuite name="t/easy/01_easy.t" errors="0" failures="0" skipped="0" tests="2" time="0.157027006149292">
+    <system-out>ok 1 - a + b = ab
```

Comment:
Should we send system-out data as stdout to Launchable? What do you think?

Reply:
Perhaps no, since it is not sent in most of the test profiles currently.